apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.html.ts

```diff
@@ -586,7 +659,7 @@ export function InstructorAssessmentInstance({
                   return html`
                     <tr>
                       <td class="text-nowrap">${row.formatted_date}</td>
-                      <td>${row.auth_user_uid}</td>
+                      <td>${row.auth_user_uid ? row.auth_user_uid : html`$mdash;`}</td>
```

Comment:
Can be slightly simplified to
```suggestion
                      <td>${row.auth_user_uid ?? html`$mdash;`}</td>
```